connectors/tests/test_cli.py

```diff
@@ -32,6 +32,7 @@ def test_main_and_kill(patch_logger, mock_responses):
     )
 
     async def kill():
+        await asyncio.sleep(0.2)
```

Comment:
@tarekziade , mock patch `run` with `kill` doesn't really start the service, so I think this may make more sense? Although I don't like it either.